### ISSUE TITLE
Evernote note stops import

### ISSUE THREAD

**Issue by crispinb** [HappenApps/Quiver#253]
The attached note (originally from the Evernote web clipper I think) will stop a Quiver import. Quiver doesn't crash, but silently stops importing with no new notebook visible in the UI. Quitting and restarting Quiver displays the new imported notebook, with all notes present up and including to this one (which is displayed sans content).

The exported note looks valid if messy in an editor, but I've only had a glance as I don't really know what to look for. (I've renamed the .enex to .txt to keep the github uploader happy)

[bad_note.txt](https://github.com/HappenApps/Quiver/files/41170/bad_note.txt)

**Comment by ylian** [comment]
Thanks. Yep, an exception was thrown. Should be trivial to fix.

**Comment by ylian** [comment]
It's due to an empty resource, no filename, no data. Might be a reminder, but not yet sure. I will fix the exception in 3.0.1.

**Comment by crispinb** [comment]
It's a pretty old note (2007!). Not sure if Evernote even had reminders then. I removed it from my export file, and the same thing happened a few notes along, so maybe it was a transient issue around the time those notes were created? Anyway presumably not worth investigating far, so long as you can get Quiver to continue the import.

Having said that, I'm assuming that the errant note will be imported rather than skipped over after the fix. If not, then you probably should consider making some kind of log reporting skipped notes available to the user.

**Comment by ylian** [comment]
It's just en empty resource, which isn't clear to me what is for. I can just skip that resource.

**Comment by ylian** [comment]
Try this build: https://dl.dropboxusercontent.com/u/3445731/Quiver%203.0.1b1.zip

I added a line to skip the empty resource.

**Comment by ylian** [comment]
This is the empty resource:

``` xml
<resource>
    <data encoding="base64"/>
    <mime>text/html</mime>
    <width>0</width>
    <height>0</height>
    <duration>0</duration>
    <resource-attributes>
        <timestamp>19700101T000000Z</timestamp>
        <attachment>true</attachment>
    </resource-attributes>
</resource>
```

This is the link to the empty resource:

``` html
<a href="http://www.1and1.com/?k_id=10219574" target="_blank">
    <en-media style="margin: 1em;" width="140" height="28" alt="Banner" border="0" align="right" type="text/html" hash="d41d8cd98f00b204e9800998ecf8427e"/>
</a>
```

The resource's type is `text/html`, maybe an iframe for an ad banner? Probably we can just ignore it.

**Comment by ylian** [comment]
Haha, the original webpage hasn't changed since 2007. The empty resource is a banner that doesn't load on the original webpage. Go to `http://javascript.crockford.com/` and scroll to the bottom.

![screen shot 2015-11-22 at 8 28 49 pm](https://cloud.githubusercontent.com/assets/984895/11328398/b89ae2cc-9157-11e5-843b-6d95dafaa8b5.png)

**Comment by crispinb** [comment]
Well it's not like much javascript-related has changed since 2007, has it? So Mr. Crockford can safely relax.

That build works well for me -- just imported the whole of the notebook that note came from without problems. It's an archive notebook with about 5000 pretty crufty old notes (lots of web clippings) so I thought it would give Quiver a good workout (up to now I've mostly used it for manually-typed notes).

Anyway thanks, that works for me.

**Comment by ylian** [comment]
Awesome. Thanks for the bug report.

**Comment by ylian** [comment]
Fixed in 3.0.1.
